Package.swift

```diff
@@ -36,7 +37,8 @@ let package = Package(
                 .product(name: "Crypto", package: "swift-crypto"),
                 .product(name: "_CryptoExtras", package: "swift-crypto"),
                 .product(name: "Logging", package: "swift-log"),
-            ],
+                .product(name:"VaporToOpenAPI",package:"VaporToOpenAPI"),
```

Comment:
This doesn't feel appropriate to add to this package as it isn't guaranteed to be used with vapor _or_ OpenAPI. As an alternative to removing it completely, could this instead live in a separate package that augments swift-webauthn for those who want this functionality.